basic/exercise1/script.js

```diff
@@ -1,30 +1,22 @@
 'use strict';
 function formatDate(date) {
-
+   
     var days = ["Monday", "Tuesday", "Wednesday", "Thursday", "Friday", "Saturday", "Sunday"];
     var weekDays = `${days[date.getDay()]}`;
-    var hours = date.getHours();
+    var hours = insertingZero(date.getHours());
     var ampm = hours >= 12 ? 'PM' : 'AM';
     hours = hours % 12;
-    if (hours < 10) {
-        hours = "0" + hours;
-    };
-
-    var minutes = date.getMinutes();
-    if (minutes < 10) {
-        minutes = "0" + minutes;
-    };
-
-    var seconds = date.getSeconds();
-    if (seconds < 10) {
-        seconds = "0" + seconds;
-    };
 
+    var minutes = insertingZero(date.getMinutes());
+    var seconds = insertingZero(date.getSeconds());
     var time = `${weekDays} ${hours} ${ampm} ${minutes}: ${seconds}`;
     return time;
-
 };
 
+ function insertingZero(timeUnit) {
```

Comment:
Please put this function before formatDate function.
It is a nice standard (but not a requirement) to have function defined in the order of how they call each other.
If function A calls function B, then function B should be defined first.
It is not required, as js execution engine will cope with it, but it is easier to read.